Cargo.toml

```diff
@@ -58,6 +58,7 @@ insta = "1.26"
 derive_builder = "0.20.0"
 wiremock = "0.6.0"
 base64 = "0.22.1"
+tracing-test = "0.2.4"
```

Comment:
Between this crate and `test_log` crate, which one do you prefer? I think test_log is slightly more popular, but I don't have a strong preference.

Reply:
I just forgot the name of the crate that you were using xD And found something online. What I like a bit more about tracing-test is that it is an "official" `tracing-xxx` crate, and also that it is composable. Instead of modifying the existing `tokio/sqlx::test` attribute, you just slap an additional attribute on top of the function.